app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,9 @@
             </intent-filter>
         </activity>
         <activity android:name=".LikedActivity"/>
+        <activity android:name=".SignUpActivity"/>
```

Comment:
Not sure if you meant to do this, but it seems like this line here has been duplicated :)